python/cuml/dask/linear_model/__init__.py

```diff
@@ -22,6 +22,7 @@
     from cuml.dask.linear_model.ridge import Ridge
     from cuml.dask.linear_model.lasso import Lasso
     from cuml.dask.linear_model.elastic_net import ElasticNet
+    from cuml.dask.linear_model.logistic_regression import LogisticRegression
```

Comment:
Regarding the namespace, I think it is a good idea to have a different namespace. Not sure if wrappers is the most intuitive for users, perhaps something like extended or so? @JohnZed @beckernick  what do you think would be a good name?

Reply:
using `cuml.dask.extended.linear_model` for now.